src/routes/LandingPage.jsx

```diff
@@ -0,0 +1,40 @@
+// React Imports
+import React from 'react';
```

Comment:
Renamed the original 'Home' component to 'Landing Page'. We have two possible 'Homes': the first place a user goes when not logged in, and the first place a user goes when they are logged in. It would be nice to have names that clearly distinguish the two.

The route to the main screen following user login was already named `home`, so this page gets renamed.

Reply:
> The route to the main screen following user login was already named `home`, so this page gets renamed.

This route is being renamed to `Clients` in PR #209  hahah.  I still think what you did is fine.  

I also plan on renaming the `forms` route to `Documents`, and then the `inbox` and `outbox` to `Messages`

Reply:
Haha well sounds like one of us will have a few merge conflicts to resolve when the other merges their PR first